skills/lark-doc/scripts/count_chars.py

```
@@ -0,0 +1,133 @@
+import sys
+import re
+import json
+import argparse
+import subprocess
+
+
+def fetch_raw_content(doc_id, identity):
+    cmd = ["lark-cli", "api", "GET",
+           f"/open-apis/docx/v1/documents/{doc_id}/raw_content", "--as", identity]
+    try:
+        out = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)